Document privileged tokens (#844)
digital-asset/docs.daml.com: docs/3.1/docs/canton/usermanual/apis.rst

Target scope can be any case-sensitive string containing alphanumeric characters, hyphens, slashes, colons and underscores.
Either the ``target-scope`` or the ``target-audience`` parameter can be configured individually, but not both at the same time.

Configuring Privileged Tokens
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Support for privileged tokens can be configured for the Ledger API exposed by the participant node as well as the Admin API exposed

> by the participant, the sequencer and the mediator nodes. The configuration follows the same pattern as for the configuration of

Reviewer: ```suggestion
by the participant, the sequencer, and the mediator nodes. The configuration follows the same pattern as for the configuration of
```

normal user tokens but with an addition of a key called ``privileged`` set to true.

> Additionally, it is possible to determine if the use of privileged tokens should result in granting of the ``participant_admin``
> or the ``wildcard`` access levels. This can be achieved by adding a definition of the ``access-level`` key and setting it to
> ``Admin`` or ``Wildcard`` respectively. The ``Admin`` is the default.

Reviewer: ```suggestion
Additionally, you can determine if the use of privileged tokens should result in granting of the ``participant_admin``
or the ``wildcard`` access levels by adding a definition of the ``access-level`` key and setting it to
``Admin`` or ``Wildcard`` respectively. The ``Admin`` is the default.
```
